app/src/main/java/com/fcaputo/parkingapp/mvp/model/SettingsModel.kt

```diff
@@ -0,0 +1,23 @@
+package com.fcaputo.parkingapp.mvp.model
+
+import com.fcaputo.parkingapp.mvp.contract.SettingsContract
+import com.fcaputo.parkingapp.utils.Constants
+import com.fcaputo.parkingapp.utils.validation.ValidationErrorType
+import com.fcaputo.parkingapp.utils.validation.ValidationResult
+
+class SettingsModel : SettingsContract.Model {
+    override fun validate(size: Int): ValidationResult {
+        return if (size == 0)
+            ValidationResult(isSuccess = false, ValidationErrorType.SIZE_IS_ZERO)
+        else
+            ValidationResult(isSuccess = true)
```

Comment:
You missed to add {} on if statement

Reply:
Is it a "must"? I mean, there's only one sentence in each branch

Reply:
Yes you always need to separate the sentences block with {}, even if you have one sentence or more than one

Reply:
If you look for a more compact version, you could:
```
return when(size) {
    0 -> ValidationResult(isSuccess = false, ValidationErrorType.SIZE_IS_ZERO)
    else -> ValidationResult(isSuccess = true)
}
```

Reply:
Ok got it